fiftyone/core/session.py

```diff
@@ -228,13 +222,7 @@ def wait(self):
     # PRIVATE #################################################################
 
     def _update_state(self):
-        # pylint: disable=attribute-defined-outside-init
-        self.state = StateDescription(
-            close=self._close,
-            dataset=self._dataset,
-            view=self._view,
-            selected=self.state.selected,
-        )
+        self.state = self.state
```

Comment:
I'm definitely confused by this

Reply:
I left a comment. I expect this to be changed, for the sake of clarity at least, when a `Session` overhaul is in order.